frontend/src/Frontend/UI/Dialogs/AddAccount.hs

```diff
@@ -0,0 +1,93 @@
+{-# LANGUAGE ConstraintKinds       #-}
+{-# LANGUAGE TupleSections #-}
+
+module Frontend.UI.Dialogs.AddAccount
+  ( uiCreateWalletOnlyAccount
+  , uiWalletOnlyAccountCreated
+  ) where
+
+import           Control.Lens
+------------------------------------------------------------------------------
+import           Reflex
+import           Reflex.Dom
+------------------------------------------------------------------------------
+import           Frontend.Wallet
+import           Frontend.Network
+import           Frontend.UI.DeploymentSettings (transactionDisplayNetwork, userChainIdSelect)
+
+import           Frontend.UI.Modal
+import           Frontend.UI.Widgets
+import           Frontend.Foundation
+
+type HasAddAccountModelCfg mConf key t =
+  ( Monoid mConf, Flattenable mConf t
+  , HasWalletCfg mConf key t
+  )
+
+uiCreateWalletOnlyAccount
+  :: forall t m model key mConf
+     . ( MonadWidget t m
+       , HasAddAccountModelCfg mConf key t
+       , HasNetwork model t
+       )
+  => model
+  -> Event t ()
+  -> m (mConf, Event t ())
+uiCreateWalletOnlyAccount model _onClose = do
+  onClose <- modalHeader $ text "Add Account"
+  (dSelectedChain, dNotes) <- modalMain $ do
+    divClass "segment modal__main transaction_details" $ do
+      elClass "h2" "heading heading_type_h2" $ text "Destination"
+      dChainId <- divClass "group segment" $ do
+        transactionDisplayNetwork model
+        userChainIdSelect model
+
+      elClass "h2" "heading heading_type_h2" $ text "Reference Data"
+      dNotes <- divClass "group segment" $
+        value <$> mkLabeledClsInput inpElem "Notes"
+
+      _ <- accordionItem' False "add-account__advanced-content" (text "Advanced") $
+        confirmButton (def & uiButtonCfg_disabled .~ pure True) "Create Vanity Account"
+
+      pure (dChainId, dNotes)
+
+  modalFooter $ do
+    onCancel <- cancelButton def "Cancel"
+    text " "
```

Comment:
Placeholder for margin?

Reply:
Since this is pre-modal redesign, perhaps we should remove all uses of this and let the redesign work fix it in CSS

Reply:
Removed `text " "` and added a ticket to fix this with the correct CSS.